gbus/tx/sagastore.go

```diff
@@ -37,8 +38,9 @@ func (store *SagaStore) scanInstances(rows *sql.Rows) ([]*saga.Instance, error)
 		var startedBySaga sql.NullString
 		var startedByMsgID sql.NullString
 		var startedByRPCID sql.NullString
+		var createdAt time.Time
```

Comment:
@kerenbe4  change this into mysql.NullTime even though the column is guaranteed  not to be null because it properly handles the parsing of the underlying type in cases in which parseTime=true or parseTime=false in the connection string and the underlying type
https://github.com/go-sql-driver/mysql/blob/15462c1d60d42ecca11d6ef9fec0b0afd5833459/nulltime.go#L20